article_store/articles/models.py

```diff
@@ -0,0 +1,84 @@
+import re
+import uuid
+
+from django.core.validators import RegexValidator
+from django.db import models
+
+
+PREVIEW = 'preview'
+PUBLISHED = 'published'
+READY = 'ready'
+
+ARTICLE_STATUSES = (
```

Comment:
these are `ARTICLE_VERSION_STATUSES`, right?

Reply:
Yes, you're right, it has now evolved into `ARTICLE_VERSION_STATUSES`.